frontend/public/components/_nav.scss

```diff
@@ -81,10 +83,10 @@ $active-border: 3px;
 
 .navigation-container__section__title {
   align-items: center;
-  color: $color-os-nav-title;
+  color: $color-pf-black-300;
   cursor: pointer;
   display: flex;
-  font-size: 15px;
+  font-size: 14px;
```

Comment:
Lines 96-100 below are no longer needed since the routing image is being replaced by an icon.

Reply:
Design team isn't moving forward with the icons right now, so I reverted those changes.